lib/prefab/config_resolver.rb

```diff
@@ -67,6 +67,16 @@ def update
 
     private
 
+    def context(properties)
```

Comment:
What is the right behavior for:
```ruby
  def filter
    Prefab.with_context({user: { id: 123, name: 'bob'}, 
                                       {team: {id: 444, name: 'Foocorp'}})
  end
  
  
  # adding some different context
  def my_method
     # will this config_client.get have access to user, team & transaction?  My expectation is yes.
     Prefab.config_client.get("flag", {transaction: { id: 8888 } })
  end
  
  # replacing some context
  def my_method
     # will this config_client.get have access to user.name My expectation is No.
     Prefab.config_client.get("flag", {user: { id: 123, email: "u@example.com } })
  end
  
    # replacing some context 
  def my_method
     # config_client.get user context would be `{ id: 455, name: "fred"} ` in my expectation
     Prefab.config_client.get("flag", {user: { id: 455, name: "fred"} })
  end
  
```

Reply:
Updated in 1a14e6b to merge JIT context with the current context. This makes your examples work as expected.